frontend/src/routes/(authenticated)/project/[project_code]/+page.svelte

```diff
@@ -245,11 +252,13 @@
           <Badge>
             <FormatRetentionPolicy policy={project.retentionPolicy}/>
           </Badge>
+          <AdminContent>
```

Comment:
Just a reminder, Kevin suggested it would make sense to show "normal" users when a project is currently being migrated. Also, `badge-warning` would probably make sense for that case.

Reply:
I think instead of a badge, a warning banner might be appropriate when a project is currently being migrated. I'll add a new div rather than turning `<AdminContent>` into a weird `{#if}` block that says "if admin status -OR- project being migrated, then show this". That would just end up awkward.

Reply:
Done.